src/main/java/com/yscope/logging/log4j1/AbstractClpIrBufferedRollingFileAppender.java

```diff
@@ -127,8 +118,9 @@ public long getCompressedSize () {
 
   @Override
   public void activateOptionsHook () throws IOException {
-    computeLogFilePath(System.currentTimeMillis());
-    clpIrFileAppender = new ClpIrFileAppender(currentLogPath, layout, useFourByteEncoding,
+    currentLogFilename = computeLogFileName(baseName, System.currentTimeMillis());
+    currentLogFilePath = computeLogFilePath(currentLogFilename);
```

Comment:
It seems like currentLogFilePath can become a local variable now since we always compute it before its use.